docs/tutorials/projects/modernize-retro-robot.md

```diff
@@ -271,17 +271,19 @@ Go to the **Setup** tab of your new robot's page and follow the steps [to instal
 {{% tab name="Builder UI" %}}
 
 Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
-Click on the **Components** subtab and navigate to the **Create component** menu.
+Click on the **Components** subtab.
 
-1. **Add the board**
+1. **Add the [board](/components/board/)**.
```

Comment:
```suggestion
1. **Add the board**.
```

nit: I'd prefer to link outside of bolded step headers / section headers. There's a good spot for it just below (my next comment).